app/src/main/java/com/fruitscale/ananas/callback/SimplerCallback.kt

```diff
@@ -0,0 +1,49 @@
+package com.fruitscale.ananas.callback
+
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.error
+import org.matrix.androidsdk.rest.callback.SimpleApiCallback
+import org.matrix.androidsdk.rest.model.MatrixError
+import java.lang.Exception
+
+/**
+ * A simple callback
+ *
+ * If an onError callback is not given it will just log the error and do nothing
+ *
+ * @param onMatrixError What to do on a Matrix error
+ * @param onNetworkError What do on a network error
+ * @param onUnexpectedError What do on an unexpected error
+ * @param onSuccess What to do on success
+ */
+class SimplerCallback<T>(private val onMatrixError: ((MatrixError) -> Unit)? = null,
```

Comment:
We use SimplerCallback instead of SimpleApiCallback so that we don't have to create a callback class each time we want to create a callback. We can just use this class and use anonymous functions